internal/daysteps/daysteps.go

```diff
@@ -10,6 +14,25 @@ var (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+dataSlise := strings.Split(data, ",")
+if len(dataSlise) != 2 {
+	return 0, 0, fmt.Errorf("Ошибка данных")
+}
+// выделяем шаги
+steps, err :=  strconv.Atoi(dataSlise[0])
+if err != nil {
+	return 0, 0, fmt.Errorf("Ошибка формата шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```

Reply:
Нужно везде поправить